styles/styles.go

```diff
@@ -2,31 +2,40 @@ package styles
 
 const (
 	// Layout Properties
-	Display         = "display"
-	Position        = "position"
-	Top             = "top"
-	Right           = "right"
-	Bottom          = "bottom"
-	Left            = "left"
-	Overflow        = "overflow"
-	OverflowX       = "overflow-x"
-	OverflowY       = "overflow-y"
-	ZIndex          = "z-index"
-	Flex            = "flex"
-	FlexDirection   = "flex-direction"
-	JustifyContent  = "justify-content"
-	AlignItems      = "align-items"
-	AlignContent    = "align-content"
-	FlexWrap        = "flex-wrap"
-	Grid            = "grid"
-	GridArea        = "grid-area"
-	GridColumn      = "grid-column"
-	GridColumnEnd   = "grid-column-end"
-	GridColumnStart = "grid-column-start"
-	GridRow         = "grid-row"
-	GridRowEnd      = "grid-row-end"
-	GridRowStart    = "grid-row-start"
-	GridTemplate    = "grid-template"
+	Display             = "display"
+	Position            = "position"
+	Top                 = "top"
+	Right               = "right"
+	Bottom              = "bottom"
+	Left                = "left"
+	Overflow            = "overflow"
+	OverflowX           = "overflow-x"
+	OverflowY           = "overflow-y"
+	ZIndex              = "z-index"
+	Flex                = "flex"
+	FlexDirection       = "flex-direction"
+	JustifyContent      = "justify-content"
+	AlignItems          = "align-items"
+	AlignContent        = "align-content"
+	FlexWrap            = "flex-wrap"
+	Grid                = "grid"
+	GridArea            = "grid-area"
+	GridAutoColumns     = "grid-auto-columns"
+	GridAutoFlow        = "grid-auto-flow"
+	GridAutoRows        = "grid-auto-rows"
+	GridColumn          = "grid-column"
+	GridColumnEnd       = "grid-column-end"
+	GridColumnGap       = "grid-column-gap"
+	GridColumnStart     = "grid-column-start"
+	GridRow             = "grid-row"
+	GridGap             = "grid-gap"
+	GridRowEnd          = "grid-row-end"
+	GridRowGap          = "grid-row-gap"
```

Comment:
Deprecated in favor of `row-gap`. Let's just change it to `RowGap` and `row-gap`.